src/modules/dpos/validator/api/index.js

```diff
@@ -3,10 +3,12 @@ import http from 'src/utils/api/http';
 import { HTTP_PREFIX } from 'src/const/httpCodes';
 import ws, { subscribe, unsubscribe } from 'src/utils/api/ws';
 import { extractAddressFromPublicKey } from '@wallet/utils/account';
+import client from 'src/utils/api/client';
+import { API_VERSION } from 'src/const/config';
 
 export const httpPaths = {
   delegates: `${HTTP_PREFIX}/accounts`,
-  votesSent: `${HTTP_PREFIX}/votes_sent`,
+  votesSent: `/api/${API_VERSION}/dpos/votes/sent`,
```

Comment:
Either we refactor all endpoints to look like this or change the `HTTP_PREFIX` value. This causes inconsistency, hence increasing the likelihood of regressions.